### PR TITLE
Added support for JSON objects/dicts without XML string. Fixed bug with preserve_entities = false

### PR DESCRIPTION
**Pull request by AndreasKrutnes**
@JonasAls89 - I had to do this because many Sesam entities lacks the XML blob from the database.

### CODE REVIEW

**Review by Ole-Morten — APPROVED** [review]
lgtm

**Comment by AndreasKrutnes** [comment]
Please merge this version whoever has access.

**Comment by AndreasKrutnes** [comment]
And please create a new release version tagged 1.0.4

File: service/service.py [hunk 1]
```diff
@@ -138,41 +138,49 @@ def xml_string_to_json():
...
+                    item["xml_as_json"] = xmltodict.parse(xmlString,encoding=xml_payload_encoding, xml_attribs=True)                                     
                 result.append(item.copy())          
      
             else:
                 xml_as_dict = {}
                 xml_as_dict = preserve_sesam_special_fields(xml_as_dict, item)
-                xml_as_dict["xml_as_json"] = xmltodict.parse(xmlString,encoding=xml_payload_encoding, xml_attribs=True)
+                if(hasXMLData):
+                    xml_as_dict["xml_as_json"] = xmltodict.parse(xmlString,encoding=xml_payload_encoding, xml_attribs=True)
+                result.append(xml_as_dict)
```
Comment (JonasAls89) [inline service/service.py]: We will try to rewrite this to yield exclusively and not write entities to a list before returning them to Sesam.

**Review by JonasAls89 — CHANGES_REQUESTED** [review]

File: service/service.py (hunk 1 above)
Comment (AndreasKrutnes) [reply service/service.py]: Changes done - new commit uploaded.

**Pull request merged**